firedrake/utility_meshes.py

```diff
@@ -198,6 +198,109 @@ def PeriodicUnitIntervalMesh(ncells):
     return PeriodicIntervalMesh(ncells, length=1.0)
 
 
+def OneElementThickMesh(ncells, L):
```

Comment:
Keep the function as is, but don't expose it in the public API.  Then just call this guy from `PeriodicRectangleMesh` in the relevant case.